--- Makefile	2007-03-09 15:41:28.000000000 +0100
+++ Makefile.oden	2008-07-29 16:00:46.000000000 +0200
@@ -19,7 +19,8 @@
 ifndef DISTBUILD
 	DEBUG_FLAGS=-ggdb -O0
 endif
-CFLAGS=-pedantic -pedantic-errors -Wall $(DEBUG_FLAGS) $(LIBPTP_INC) $(HAVE_GTK) $(HAVE_READLINE)
+#CFLAGS=-pedantic -pedantic-errors -Wall $(DEBUG_FLAGS) $(LIBPTP_INC) $(HAVE_GTK) $(HAVE_READLINE)
+CFLAGS= $(RPM_OPT_FLAGS) -Wall $(DEBUG_FLAGS) $(LIBPTP_INC) $(HAVE_GTK) $(HAVE_READLINE)
 LDLIBS= -lusb $(LIBPTP_LIB)
 
 EXECUTABLE  := capture
